wandb/sdk/data_types/base_types/wb_value.py

```diff
@@ -13,16 +13,20 @@
 def _server_accepts_client_ids() -> bool:
     from wandb.util import parse_version
 
-    # First, if we are offline, assume the backend server cannot
-    # accept client IDs. Unfortunately, this is the best we can do
-    # until we are sure that all local versions are > "0.11.0" max_cli_version.
-    # The practical implication is that tables logged in offline mode
-    # will not show up in the workspace (but will still show up in artifacts). This
-    # means we never lose data, and we can still view using weave. If we decided
-    # to use client ids in offline mode, then the manifests and artifact data
-    # would never be resolvable and would lead to failed uploads. Our position
-    # is to never lose data - and instead take the tradeoff in the UI.
-    if util._is_offline():
+    # There are versions of W&B Server that cannot accept client IDs. Those versions of
+    # the backend have a max_cli_version of less than "0.11.0." If the backend cannot
+    # accept client IDs, manifests and artifact data would never be resolvable and lead
+    # to failed uploads. Our position is to never lose data - and instead take the
+    # tradeoff in the UI. The results in tables not displaying media correctly, but
+    # the table can still be accessed via the .artifact op.
+    #
+    # The latest SDK version that is < "0.11.0" was released on 2021/06/29.
+    # AS OF NOW, 2024/11/06, we assume that the large majority server deployments accept
+    # client IDs.
+    #
+    # If there are any users with issues on an older backend, they can use offline_legacy
+    # mode for legacy behavior.
+    if util._is_offline_legacy():
```

Comment:
see my comment in the settings file, this is too aggressive for what you are trying to achieve, you just want a specific setting that will allow safely to bypass all these old checks and give this setting to our users.

offline is pretty fundamental mode of operation in the client so it is a bad idea to muck with it.